Libraries/pbxspec/Sources/Manager.cpp

```diff
@@ -86,9 +86,13 @@ findSpecification(std::vector<std::string> const &domains, std::string const &id
 {
     typename T::vector vector = findSpecifications <T> (domains, type);
```

Comment:
Aha, looks like you're right. I think the problem comes from `Manager::inheritSpecification`, in the block that says this:

```
        /*
         * Search the specified domain then in any domain. Some specifications inherit
         * from domain/identifier pairs that don't exist in practice, but by using any
         * domain they can successfully inherit from something potentially relevant.
         */
```

I suppose we should do two things:
* make sure that if a domain is specified, and an identifier in that domain exists, we pick that one first instead of picking the one from `<<any domain>>`;
* and then, if we have to rely on the wildcard domain, enforce a check to make sure that we don't inherit from ourselves (which is where the failure I'm trying to fix comes from.